HousingRepairsOnlineApi/Helpers/NotificationConfiguration/BaseNotificationConfigurationProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Ardalis.GuardClauses;
+using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.UseCases;
+
+namespace HousingRepairsOnlineApi.Helpers.SendNotifications;
+
+public class BaseNotificationConfigurationProvider
```

Comment:
Nitpick: Given this doesn't provide functionality of its own, consider making it `abstract`.

Reply:
Alternatively, add a [default interface implementation](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/default-interface-methods) and remove this base class. As per my other [comment](https://github.com/Newarkandsherwood/housing-repairs-online-api/pull/36/files/7f627cd898da4ee9ad440dc7403c535129269bdd..c74a355a11258c80e69416d74535c8242da831ae#r1026226161), `GetImageLink` doesn't seem related to configuration

Reply:
I've made it abstract so now it just sets the initial properties in the constructor. I wasn't sure a default interface implementation could be used here. You can have methods, but can you define the actual properties? I'll have another look once I've made these initial changes.